testsuite/objects/__init__.py

```diff
@@ -109,6 +109,22 @@ def asdict(self):
         return {"name": self.name, **asdict(self.value)}
 
 
+@dataclass
+class ExtendedProperty(Property):
+    """
+    Dataclass extending Property class adding optional `overwrite` feature
+    used in extended_properties functionality in Identity section.
+    """
+
+    overwrite: Optional[bool] = None
+
+    def asdict(self):
+        """Override `asdict` function."""
+        if self.overwrite is not None:
```

Comment:
Shouldnt custom asdict take care of this?

Reply:
I inherit from a class that defines `asdict()` override method so I need to redefine it, otherwise the parent method would be used.
But yes, the custom `asdict()` automatically do not include None variables.